src/hotspot/share/gc/shenandoah/mode/shenandoahGenerationalMode.cpp

```diff
@@ -32,7 +32,7 @@
 
 void ShenandoahGenerationalMode::initialize_flags() const {
 
-#if !(defined AARCH64 || defined AMD64 || defined IA32 || defined PPC64)
+#if !(defined AARCH64 || defined AMD64 || defined IA32 || defined PPC64 || RISCV64)
   vm_exit_during_initialization("Shenandoah Generational GC is not supported on this platform.");
```

Comment:
Should RISCV64 use the same style of macro test as the other architectures?  "|| defined RISCV64"

Reply:
Good catch. I think it's mis-typed. Fixed. Thank you.